### ISSUE TITLE
"protein binding" via PAINT/IBA

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-annotation#2097]
There are currently 5686 annotations for GO:0005515 protein binding (exact match) evidenced by IBA (seen via 
[protein_binding_PAINT.txt](https://github.com/geneontology/go-annotation/files/2441772/protein_binding_PAINT.txt)

AmiGO)
For obvious reasons, I am pretty sure that this is not intended propagation.
Perhaps screening GO:0005515 could be folded into QC/QA in the PAINT pipeline?

@tonysawfordebi - perhaps this is another thing that has been going on for a while and we haven't noticed as they were blocked by GOA QC?

**Comment by hattrill** [comment]
It's just that I've never seen them before (it gets flagged in our pre-release checks, so I would have seen them)........although we did just change something with our import pipeline (because of the change in the PAINT "with" column), so perhaps we've broken a block that we had that stopped GO:0005515 without an IPI?
I'll have to grub about in our load files.

**Comment by pgaudet** [comment]
We never did that before. There is a new curator who has been given incorrect guidelines, we're in the process of fixing annotations. Sorry about that.

**Comment by huaiyumi** [comment]
The number is probably inflated caused by the data loading issue. Based on the database history, paint had 137 such annotations. 12 were obsoleted. Currently there are 127 such annotations contributed by Marc, Michael (the new curator) and me.

**Comment by hattrill** [comment]
How long before this fix comes through? I can see 5710 annotations direct to protein binding in AmiGO GO:0005515.

**Comment by marcfeuermann** [comment]
We have removed all the unfortunate "protein binding" propagations in PAINT. This means that they should disappear in the close future :-)
I'll keep the ticket open until this will be totally resolved.

**Comment by pgaudet** [comment]
One family left - PTHR10336. Will ask Anushya to remove the annotations (as we have removed the ability to view/add these annotations in PAINT). 

Thanks, Pascale

**Comment by pgaudet** [comment]
Annotations were removed in July. All protein binding annotations should be gone by the next AmiGO release.
